website/docs/reference/opspec/op.yml/predicate.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Predicate
+---
+
+A predicate is a condition that evaluates to a boolean value, used for conditional logic. The predicate is an object with a single key defining the type of condition.
+
+## `eq`
+
+An [array](../types/array). When all items in the array are equivalent, the predicate is true.
+
+```yml
```

Comment:
I really like having examples for each documented thing.  I think it might make it slightly more accessible if the examples followed a standard format, where they are labelled as an example, the purpose is briefly described, the example is presented and the expected behavior is summarized.

It would add some length to the docs, but I think it'd add a lot of clarity to the examples